# HG changeset patch
# User Christoph Junghans <christoph.junghans@googlemail.com>
# Date 1268811801 -3600
# Node ID fce276b5e0465d7558dddcf0d3310ecbdf2deb7c
# Parent  9c22ee7672d6924a066f46c1b0e7bad076641cc5
added ACLOCAL_AMFLAGS back to Makefile.am

diff -r 9c22ee7672d6 -r fce276b5e046 Makefile.am
--- a/Makefile.am	Tue Mar 16 22:30:42 2010 +0100
+++ b/Makefile.am	Wed Mar 17 08:43:21 2010 +0100
@@ -1,3 +1,5 @@
+ACLOCAL_AMFLAGS = -I config
+
 SUBDIRS = src include scripts
 
 EXTRA_DIST = CHANGELOG NOTICE LICENSE
diff -r 9c22ee7672d6 -r fce276b5e046 src/libboost/Makefile.am
--- a/src/libboost/Makefile.am	Tue Mar 16 22:30:42 2010 +0100
+++ b/src/libboost/Makefile.am	Wed Mar 17 08:43:21 2010 +0100
@@ -1,3 +1,5 @@
+ACLOCAL_AMFLAGS = -I config
+
 SUBDIRS = libs boost
 
 pkgconfigdir = $(libdir)/pkgconfig
